internal/indexer/moniker.go

```diff
@@ -53,7 +53,8 @@ func joinMonikerParts(parts ...string) string {
 // identifier (either a range or a result set identifier). This will also emit links between
 // the moniker vertex and the package information vertex representing the dependency containing
 // the identifier.
-func (i *Indexer) emitImportMoniker(sourceID uint64, p *packages.Package, obj types.Object) {
+// TODO THINGS
```

Comment:
What things?

Reply:
oops, those things were to-done :laughing: